queries/proto/highlights.scm

```diff
@@ -0,0 +1,48 @@
+[
+  "syntax"
+  "package"
+  "option"
+  "import"
+  "service"
+  "rpc"
+  "returns"
+  "message"
+  "enum"
+  "oneof"
+  "repeated"
+  "reserved"
+  "to"
+] @keyword
+
+[
```

Comment:
`"\"proto3\""` could be `@string`, `=` and `;` could be highlighted.

Enum identifiers could be highlighted as `@constant` but not a must

Reply:
Pushed all suggestions except for `=`. Do you think we should use operator for that?

Reply:
I think `@operator` should be good for that